### ISSUE TITLE
more than one testimonials

### ISSUE THREAD

**Issue by fdiblen** [research-software-directory/landing-page#42]
It might be useful if more than one testimonials can be added.

**Comment by jspaaks** [comment]
Thanks for the feedabck Faruk. I'm currently refactoring the schema (without knowing much about schemas, it has to be said :). The relevant section there is: 
https://github.com/research-software-directory/backend/blob/87f83cbb27ae46d9f7ad6a75d239e7e1f63eee7c/schema/software.json#L210-L237

**Comment by sverhoeven** [comment]
It has been defined as array, see https://github.com/research-software-directory/backend/blob/4af3e4d7531ad37a2b1223369faa87665749630b/schemas/software.json#L569-L597

**Comment by sverhoeven** [comment]
Frontend needs to loop over testimonials, see https://github.com/research-software-directory/frontend/blob/b9e745e63462754faecbf644a4d3457b7689ed4f/templates/software/projects.html#L9-L21

**Comment by jspaaks** [comment]
@sverhoeven I think you're suggesting to show all testimonials and all projects, right? I propose to show just one, chosen at random. I think it would be even better if we can link it to the project, but I don't think that;s something the data model supports, so we'll leave that for post-``v2``.

**Comment by sverhoeven** [comment]
Yes, as projects and testimonials are not combined anymore, I would like show all projects and all testimonials.

I think a testimonial is more likely to occur in one of our projects, but a testimonial outside our projects is even better. Showing a testimonial tied to a project will lessen the impact of a testimonial without a project. So I would not like to attach a project to a testimonial.

Let's not try do anything random, the site should be predictable, so just pick the first or last testimonial.

**Comment by sverhoeven** [comment]
For now the first testimonial is rendered by commit https://github.com/research-software-directory/frontend/commit/0ca8b3fbe04a629342fc1880bf075d86bd30e708 , but more can be stored.

**Comment by jspaaks** [comment]
Seems like a good compromise to me, we'll figure out what to do with multiple testimonials once we start having cases that have that.
